src/lib/runtime_config/runtime_config.ml

```diff
@@ -1686,6 +1686,12 @@ let slot_tx_end, slot_chain_end =
   in
   (f (fun d -> d.slot_tx_end), f (fun d -> d.slot_chain_end))
 
+let hard_fork_genesis_slot_delta t =
+  let open Option.Let_syntax in
+  let%bind daemon = t.daemon in
+  let%map delta = daemon.hard_fork_genesis_slot_delta in
```

Comment:
I was going to suggest we change t.daemon.hard_fork_genesis_slot_delta type to be `Global_slot_span.t` so this conversion is not needed every time we need this value. 

However it seems there's some obscure json conversion there. I guess we can add a note there for follow-up if there's ever going to be refactor around there.